hack/make/tests/e2e.mk

```diff
@@ -64,20 +64,28 @@ test/e2e/cloudnative/codemodules: manifests/crd/helm
 
 ## Runs CloudNative codemodules-with-proxy e2e test only
 test/e2e/cloudnative/codemodules-with-proxy: manifests/crd/helm
-	go test -v -tags "$(shell ./hack/build/create_go_build_tags.sh true)" -timeout 20m -count=1  ./test/scenarios/istio -args --feature "codemodules-with-proxy" $(SKIPCLEANUP)
+	go test -v -tags "$(shell ./hack/build/create_go_build_tags.sh true)" -timeout 20m -count=1  ./test/scenarios/istio -args --feature "codemodules-with-proxy$$" $(SKIPCLEANUP)
 
 ## Runs CloudNative codemodules-with-proxy-custom-ca e2e test only
-test/e2e/cloudnative/codemodules-with-custom-ca: manifests/crd/helm
-	go test -v -tags "$(shell ./hack/build/create_go_build_tags.sh true)" -timeout 20m -count=1  ./test/scenarios/istio -args --feature "codemodules-with-proxy-custom-ca" $(SKIPCLEANUP)
+test/e2e/cloudnative/codemodules-with-proxy-custom-ca: manifests/crd/helm
+	go test -v -tags "$(shell ./hack/build/create_go_build_tags.sh true)" -timeout 20m -count=1  ./test/scenarios/istio -args --feature "codemodules-with-proxy-custom-ca$$" $(SKIPCLEANUP)
```

Comment:
why the `$$`

Reply:
`$` regex symbol matching the null string at the end of a line - I want single test to be executed only (`A$` for A only, without AB, AC)

`$$` because `$` is "special character" for make comamnd (`$<`, `$%`,...) and make tries to parse `$"` instead of `$` and `"` separately.

In case of single `$`: 
```
bash: -c: line 1: unexpected EOF while looking for matching `"'
```

Reply:
why not rename the 
- `codemodules-with-proxy-and-auto-ag-cert` to `codemodules-auto-ag-cert-with-proxy`
- `codemodules-with-proxy-custom-ca-auto-ag-cert` to `codemodules-auto-ag-cert-with-proxy-custom-ca`

so we don't need the magic :D